### PR TITLE
update run.yaml

### PR DESCRIPTION
**Pull request by adisky**
trying optimization, to make build not fail

### CODE REVIEW

**Comment by ramineni** [comment]
@adisky one question , why are we removing KUBE_FASTBUILD=true , I believe then it will compile for all other architectures as well, which is not required?

File: playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml [hunk 1]
```diff
@@ -116,14 +115,16 @@
           # Run e2e test using local deployment/provider
           # Must run kubetest under kubernetes root directory
           cd '{{ k8s_src_dir }}'
+          
+          #temproary change to solve memory issue
```
Comment (moo-ai) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]:
```suggestion
          # temproary change to solve memory issue
```

Comment (wangxiyuan) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]:
```suggestion
          # temproary change to solve memory issue
```

**Comment by adisky** [comment]
> @adisky one question , why are we removing KUBE_FASTBUILD=true , I believe then it will compile for all other architectures as well, which is not required?

@ramineni added

File: playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml [hunk 2]
```diff
@@ -116,14 +116,16 @@
           # Run e2e test using local deployment/provider
           # Must run kubetest under kubernetes root directory
           cd '{{ k8s_src_dir }}'
+
```
Comment (moo-ai) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: This line contains trailing blanks.

Comment (wangxiyuan) [inline playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml]: This line contains trailing blanks.

**Pull request merged**